data/reusables/actions/runner-name-description.md

```diff
@@ -1 +1 @@
-The name of the runner executing the job.
+The name of the runner executing the job. Also the name may not be unique in a workflow run as runners at repository and organization level could use the same name.
```

Comment:
Technically, I believe it also could be an `enterprise level` runner. I don't know if that requires conditional markup or not.